test/data/manifests/fedora_34-x86_64-fedora_iot_commit-boot.json

```diff
@@ -11152,9 +13890,9 @@
       "elfutils-libs-0.186-1.fc34.x86_64",
       "expat-2.4.1-1.fc34.x86_64",
       "fedora-gpg-keys-34-2.noarch",
+      "fedora-release-34-39.noarch",
       "fedora-release-common-34-39.noarch",
-      "fedora-release-identity-iot-34-39.noarch",
-      "fedora-release-iot-34-39.noarch",
```

Comment:
This looks suspicious. The `fedora-release-iot` should be probably still installed...